apps/web/src/components/PoolCandidatesTable/PoolCandidateDocument.tsx

```diff
@@ -126,143 +126,157 @@ const PoolCandidateDocument = React.forwardRef<
       <div ref={ref}>
         <div className="print-container">
           {candidates &&
-            candidates.map((candidate) => {
+            candidates.map((candidate, index) => {
               return (
-                <div key={candidate.id} className="page-wrapper">
-                  <div className="page-section">
+                <>
+                  <div key={candidate.id} className="page-wrapper">
```

Comment:
Oops, don't forget to move the key up the top element in the map.

Reply:
Nice catch! Should be fixed by [a28a984](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5705/commits/a28a9848a3844db615243ad9cdc13734873816f6)